src/mesh.rs

```diff
@@ -138,22 +135,16 @@ pub fn load_mesh(
                 ))
             }
         }
-        Some("stl" | "STL") => {
-            debug!("load stl: path = {file_string}");
-            load_stl(&fetch_or_read(file_string)?, scale, opt_color, group)
-        }
-        Some("dae" | "DAE") => {
-            debug!("load dae: path = {file_string}");
```

Comment:
I plan to support fallback file type detection if the extension is not matched as assimp does, so remove the extension checks here except for obj.

Reply:
And it is now supported: https://github.com/openrr/mesh-loader/pull/34